src/app/settings.ts

```diff
@@ -193,6 +193,18 @@ export class Settings {
     return null;
   }
 
+  StringToArray(str): string {
```

Comment:
I like the fact that you did this function. But the function/methods name should match what it's doing. It's not doing a `StringToArray`, it's actually generating the title.